content/docs/en/elements/components/tab-view.md

```diff
@@ -35,7 +35,7 @@ contributors: [MisterBrownRSA, rigor789, eddyverbruggen, ikoevska, kharysharpe]
   </TabViewItem>
 </TabView>
 ```
-**NOTE:** Use images for tab icons instead of icon fonts.
+**NOTE:** Use images for tab icons instead of icon fonts. Because this is a native component, to control the size of the icons you should use resources as defined in https://github.com/nativescript-vue/nativescript-vue.org/blob/master/content/docs/en/elements/components/tab-view.md
```

Comment:
This doesn't make any sense. You are referring to the page you're currently on (only to its source in GitHub). Is there a better link to use here?